### ISSUE TITLE
Неверная орфография

### ISSUE THREAD

**Issue by danttur** [dwayned84/testing12#27]
Browser: Mozilla 232323
Google version 1.0

Шаги
Пользователь переходит на главную страницу:http://www.google.com

Фактический результат Элемент консоли Hopping, элементы поиска Langauge tools, Gooogle Search, I'm Feeling Unlucky
Ожидаемый результат Элемент консоли Hopping должен быть Shopping, элементы поиска Langauge tools, Gooogle Search, I'm Feeling Unlucky должны быть Language Tools, Google search, I'm feeling lucky.

**Comment by dwayned84** [comment]
@danttur Отлично! Немного ожидаемый результат скомкан и трудно читаемый, но в целом структура тикета ок. Забыли указать название компонента в названии тикета и закинуть тикет на кого-то

**Comment by danttur** [comment]
> @danttur Отлично! Немного ожидаемый результат скомкан и трудно читаемый, но в целом структура тикета ок. Забыли указать название компонента в названии тикета и закинуть тикет на кого-то

Спасибо! Немного переделал, такая структура тикета правильна?

**Comment by dwayned84** [comment]
@danttur Да, только у вас тикет про орфографию, а вы туда запихнули проблему с Done Done, что относиться к проблема с браузером Мозилла, а мы работаем на Гугл)

**Comment by danttur** [comment]
ok
